playbooks/ai-on-openlab/pre.yaml

```diff
@@ -0,0 +1,93 @@
+- name: AI On Openlab Pre Job
+  hosts: all
+  become: yes
+  tasks:
+    - name: Set Default Hardware type Parameters
+      set_fact:
+        user_file: "user_data"
+      when: not is_gpu_enabled
+
+    - name: Split Hardware type Parameters for GPU
+      set_fact:
+        user_file: "user_data_gpu"
+      when: is_gpu_enabled
+
+    - name: Shape login Demo User
+      shell:
+        cmd: |
+          set -x
+          set -e
+
+          useradd demo -p 'demo'
+          echo 'demo:demo' | chpasswd
+          usermod -s /bin/bash demo
+          mkdir /home/demo
```

Comment:
nit: `mkdir -p /home/demo`

Even I know the '/home' dir is exists in general case. : )